yarn-project/end-to-end/src/sample-dapp/index.mjs

```diff
@@ -33,13 +33,20 @@ async function showPrivateBalances(pxe) {
 
 // docs:start:mintPrivateFunds
 async function mintPrivateFunds(pxe) {
-  const [owner] = await getInitialTestAccountsWallets(pxe);
-  const token = await getToken(owner);
+  const [ownerWallet] = await getInitialTestAccountsWallets(pxe);
+  const token = await getToken(ownerWallet);
 
   await showPrivateBalances(pxe);
 
   const mintAmount = 20n;
-  await mintTokensToPrivate(token, owner, owner.getAddress(), mintAmount);
```

Comment:
The issue here was missing import of `mintTokensToPrivate`. This was not caught by the build process for some reason (I guess because it's .mjs)? Since this is an example app I should have not use that import in the first place as we just want to refer aztec.js so I just imported the relevant code.